.drone.yml

```diff
@@ -86,15 +86,40 @@ data:
         event:
           - tag
 
-    # bot-twitter:main and frontend:$MAIN_COMMIT_SHA
+    # frontend-production:$RELEASE_TAG
+    - name: publish-frontend-production-on-tag
+      repository: ghcr.io/sidestream-tech/auction-ui/frontend-production
```

Comment:
Do we keep old registry because it allows us to keep images private?

Reply:
No, just a value that was not yet updated. Pointed it to use new repo name!